v21.2/deploy-app-vercel.md

```diff
@@ -17,6 +22,54 @@ Before starting the tutorial, do the following:
 
 1. Create a [Vercel](https://vercel.com/signup) account.
 
+<section class="filter-content" markdown="1" data-scope="browser">
+
+## Step 1. Get the code
+
+1. Create a [GitHub](https://github.com/) account if you haven't already.
+1. In GitHub, [create your own fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) of CockroachDB's [`prisma-examples` repo](https://github.com/cockroachdb/prisma-examples).
+
+## Step 2. Create a Vercel project
+
+1. From the [Vercel dashboard](https://vercel.com/dashboard), click **Add new...** > **Project**.
+1. Select the `prisma-examples` repository you forked in [Step 1](#step-1-get-the-code) to import.
+1. From the **Framework preset** dropdown, select **Next.js**.
+1. In the **Root directory** field, click **Edit**.
+1. Select the **typescript** > **rest-nextjs-api-routes** directory.
+1. Click **Continue**.
+1. Open the **Build and Output Settings** and toggle the build command's **Override** switch to on.
+1. Enter `yarn prisma db push && yarn next build` as the **Build command**.
+1. Click **Deploy**.
+
+    Your deployment will fail until you integrate CockroachDB in the next step, so you can leave this screen without waiting for it to finish.
+
+## Step 3. Integrate CockroachDB
+
+1. Navigate to the [CockroachDB page](https://vercel.com/integrations/cockroachdb) of Vercel's integration marketplace.
+1. Click **Add Integration**.
+1. Select your Vercel account and click **Continue**.
+1. Select either **All Projects** or the project you just created and click **Continue**.
```

Comment:
I'd prefer to recommend to select only the project that the user just created. That will reduce the chance of trying to link other projects that the user has.